nix/overlay.nix

```diff
@@ -193,16 +193,4 @@ in
     '';
     subPackages = [ "cmd/speeve" ];
   };
-
-  vast-ui = nnbp.buildYarnPackage {
```

Comment:
We should remove `nix-npm-buildpackage` from the flack and update the lockfile as well. I can take care of that if you want.